front/lib/utils/apps.ts

```diff
@@ -130,27 +130,29 @@ async function updateDatasets(
       }
 
       // Now update the dataset in front if it exists, or create one
-      const dataset = existingDatasets.find(
-        (d) => d.name === datasetToImport.name
-      );
-      if (dataset) {
-        if (
-          !_.isEqual(dataset.schema, datasetToImport.schema) ||
-          dataset.description !== datasetToImport.description
-        ) {
-          await dataset.update({
+      if (datasetToImport.schema) {
```

Comment:
We don't update/create dataset in front if we don't have a schema. It will only be stored in core.